pkg/prometheus/promcfg_test.go

```diff
@@ -8638,6 +8695,15 @@ func TestOTLPConfig(t *testing.T) {
 			},
 			golden: "OTLPConfig_Config_translation_strategy_with_unsupported_version.golden",
 		},
+		{
+			name:    "Config Legacy nameValidationScheme with OTLP translation strategy",
```

Comment:
we need another test case with translation strategy == underscore with suffixes

Reply:
Done, pls review